crates/forge/bin/cmd/soldeer.rs

```diff
@@ -0,0 +1,105 @@
+use clap::Parser;
+use eyre::Result;
+
+extern crate soldeer_lib;
```

Comment:
This is not needed

Reply:
this has been solved, waiting for all the comments to have a resolution before pushing a new version